cosmos/cache.py

```diff
@@ -171,3 +173,64 @@ def _copy_partial_parse_to_project(partial_parse_filepath: Path, project_path: P
 
     if source_manifest_filepath.exists():
         shutil.copy(str(source_manifest_filepath), str(target_manifest_filepath))
+
+
+def get_cache_profile_version_path(current_version: str) -> Path | None:
+    """
+    Check if profile version and correspond dbt profile yml exit in metadata.json and return the yml path or None
+    """
+    if not dbt_profile_cache_path:
```

Comment:
This should be optional. We should not expect users to have to set this path - we already have a default `cache_dir`, why not reuse it?

Reply:
We can make it optional, I wanted to keep the profile caching-related files in one place. Shall we keep the profile related files in cache_dir / profile ?